web/themes/joinup/prototype/sass/components/_tab.sass

```diff
@@ -7,101 +7,97 @@
 // Component
 // --------------------------------------------------
 .tab
-  background-color: $color-grey-light
-
-.tabs
-  border-bottom: 1px solid $color-grey-light
+  background-color: transparent
```

Comment:
Some spacing might be useful.

![download](https://user-images.githubusercontent.com/1923476/28713019-b1c502ce-738d-11e7-89dc-7c884e2706c2.png)